core/variant_call.cpp

```diff
@@ -1360,6 +1360,8 @@ Variant Variant::construct(const Variant::Type p_type, const Variant **p_args, i
 			case BASIS: return (Basis(p_args[0]->operator Basis()));
 			case TRANSFORM:
 				return (Transform(p_args[0]->operator Transform()));
+			case TRANSFORM2D:
+				return (Transform2D(p_args[0]->operator Transform2D()));
```

Comment:
It should come after `case VECTOR3I` to keep the same order as in `core/variant.h`.

Reply:
@akien-mga  done 👍